src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -105,15 +130,25 @@ class ComplianceCard extends Component {
                             )
                         }
                         {complianceFetchStatus === 'pending' && (<Loading />)}
+                        <div className="ins-c-compliance__row">
+                            <div className="ins-c-compliance__row-item">
+                            </div>
+                            <div className="ins-c-compliance__row-item">
+                                <Button
+                                    className="ins-c-compliance__policy-link"
+                                    component="a"
+                                    href={ `/${UI_BASE}/compliance/policies/` }
+                                    variant="link"
+                                    isInline
+                                >
+                                    {complianceFetchStatus === 'fulfilled' && Array.isArray(complianceSummary.data) &&
+                                    complianceSummary.data.length > 1 ? ` ${complianceSummary.data.length} ` : ' '}
+                                    more compliance policies
```

Comment:
Need to build this string out in messages.js, gotta keep up with translations 🦹‍♀️